### PR TITLE
Channel ingester Nitro

### PR DESCRIPTION
**Pull request by Spartakovic**
Added Nitro channel and masterbrands ingester and task

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/bbc/nitro/GlycerinNitroChannelAdapter.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    }
+
+    public static GlycerinNitroChannelAdapter create(Glycerin glycerin) {
+        return new GlycerinNitroChannelAdapter(glycerin);
+    }
+
+    @Override
+    public ImmutableSet<Channel> fetchServices() throws GlycerinException {
+        ServicesQuery servicesQuery = ServicesQuery.builder()
+                .withPageSize(MAXIMUM_PAGE_SIZE)
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/GlycerinNitroChannelAdapter.java]: Don't you to paginate this request?

File: src/test/java/org/atlasapi/remotesite/bbc/nitro/GlycerinNitroChannelAdapterTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GlycerinNitroChannelAdapterTest {
```
Comment (Dalamar42) [inline src/test/java/org/atlasapi/remotesite/bbc/nitro/GlycerinNitroChannelAdapterTest.java]: You should add tests for `fetchMasterbrands` as well

File: src/main/java/org/atlasapi/remotesite/bbc/nitro/channels/NitroChannelHidrator.java [hunk 3]
```diff
@@ -62,8 +63,16 @@ public NitroChannelHidrator() {
...
     }
 
+    private final Predicate<Channel> inServiceTable() {
+        return IN_SERVICE_TABLE;
+    }
+
+    private final Predicate<Channel> inMasterbrandTable() {
+        return IN_MASTERBRAND_TABLE;
+    }
+
```
Comment (atish-metabroadcast) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/channels/NitroChannelHidrator.java]: why are you wrapping the above?  Surely these are not relevant?

Comment (MartinsIrbe) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/channels/NitroChannelHidrator.java]: @atish-metabroadcast according to our Java style guide (Constant) :)

**Pull request closed without merging**